Example/Messaging/Sample/Sample/ContentView.swift

```diff
@@ -81,40 +81,47 @@ struct ContentView: View {
       }
     }
   }
+
   func getToken() {
-    InstanceID.instanceID().instanceID { (result, error) in
-      if result != nil && error == nil {
-        self.identity.token = result?.token ?? ""
-        self.identity.instanceID = result?.instanceID ?? ""
+    InstanceID.instanceID().instanceID { result, error in
+      if error == nil {
+        print("Failed getting iid and token: ", error ?? "")
+        return
       }
+      guard let result = result else {
```

Comment:
I think it can even be combined with lines 87-89 like:

```
guard 
  let let result = result,
  error == nil
else {
  print()
  return
}
```